index.html

```diff
@@ -11,6 +11,7 @@
 </head>
 
 <body>
+    <h1>hey</h1>
     <script src="./assets/js/index.js"></script>
```

Comment:
Can we move this `./assets/js/index.js` after the body tag

Reply:
Hey @xDAnkit, yes I'll be moving it to the bottom

Reply:
Thank you @98arun for considering the following changes